backend/corpora/common/corpora_config.py

```diff
@@ -11,10 +11,24 @@ def __init__(self, *args, **kwargs):
 
     def get_defaults_template(self):
 
+        # Set collections_base_url
+        RDEV_PREFIX = os.environ.get("REMOTE_DEV_PREFIX")
```

Comment:
these values should be lower case since they are not CONSTANTS

Reply:
good point